src/settingsScreen.js

```diff
@@ -357,165 +478,134 @@ class SettingsScreen {
                 MaximumValue: 4,
                 Units: 5,
             };
-            await match.accessible.Text(SettingsLabel.CorrectionRange).tap();
+            await this.CorrectionRangeLabel().tap();
             if (preMeal) {
                 await match.accessible.Label('Pre-Meal').tap();
                 await match.accessible.PickerItem(2, `${preMeal.max}`).tap();
                 await match.accessible.PickerItem(2, `${preMeal.min}`).atIndex(glucosePreMealOverridePickerColumns.MinimumValue).tap(); //sets min
             }
-            await match.accessible.Label(Label.Save).tap();
-            await this._exitSetting();
+            await match.accessible.Label(text.general.Save).tap();
+            await _exitSetting();
         }
-
     }
     /**
-     * @name settings.ClosedLoop
      * @summary turn on closed loop mode
      */
     async ClosedLoop() {
```

Comment:
[nit] Since this is taking an action to turn on closed loop mode, might want to consider making the function name action oriented (e.g. `SetClosedLoop` or something like that). If a function is mutating state (changing something) then I find it easier to understand if the function name is an action (e.g. Set, Do, Load, Close, Open, etc.).